frontend/app/src/composables/balances/breakdown.ts

```diff
@@ -4,7 +4,7 @@ import type { MaybeRef } from '@vueuse/core';
 import type { AssetBreakdown } from '@/types/blockchain/accounts';
 
 interface UseBalancesBreakdownReturn {
-  assetBreakdown: (asset: string) => ComputedRef<AssetBreakdown[]>;
+  assetBreakdown: (asset: string, isLiability?: boolean) => ComputedRef<AssetBreakdown[]>;
```

Comment:
Here also split into `assetBreakdown` and `liabilityBreakdown` and we should make sure that it only tracks liabilities.